lib/Net/DRI/DRD/UniRegistry/DM.pm

```diff
@@ -0,0 +1,117 @@
+## Domain Registry Interface, UniRegistry::DM Driver
+##
+## Copyright (c) 2021 Patrick Mevzek <netdri@dotandco.com>. All rights reserved.
+##           (c) 2021 Michael Holloway <michael@thedarkwinter.com>. All rights reserved.
+##           (c) 2021 Paulo Jorge <paullojorgge@gmail.com>. All rights reserved.
+##
+## This file is part of Net::DRI
+##
+## Net::DRI is free software; you can redistribute it and/or modify
+## it under the terms of the GNU General Public License as published by
+## the Free Software Foundation; either version 2 of the License, or
+## (at your option) any later version.
+##
+## See the LICENSE file that comes with this distribution for more details.
+####################################################################################################
+
+package Net::DRI::DRD::UniRegistry::DM;
+
+use strict;
+use warnings;
+
+use base qw/Net::DRI::DRD::UniRegistry::UniRegistry/;
+
+use DateTime::Duration;
+
+=pod
+
+=head1 NAME
+
+Net::DRI::DRD::UniRegistry::DM - UniRegistry::DM Driver for Net::DRI
+
+=head1 DESCRIPTION
+
+Additional domain extension UniRegistry::DM New Generic TLDs
+
+UniRegistry::DM utilises a dedicated server and the following standard extensions. Please see the test files for more examples.
+
+=head2 Standard extensions:
+
+=head3 L<Net::DRI::Protocol::EPP::Extensions::secDNS> urn:ietf:params:xml:ns:secDNS-1.1
+
+=head3 L<Net::DRI::Protocol::EPP::Extensions::GracePeriod> urn:ietf:params:xml:ns:rgp-1.0
+
+=head3 L<Net::DRI::Protocol::EPP::Extensions::LaunchPhase> urn:ietf:params:xml:ns:launch-1.0
+
+=head3 L<Net::DRI::Protocol::EPP::Extensions::IDN> urn:ietf:params:xml:ns:idn-1.0
+
+=head2 Custom extensions:
+
+=head3 L<Net::DRI::Protocol::EPP::Extensions::CentralNic::Fee> urn:centralnic:params:xml:ns:fee-0.7
+
+
+=head1 SUPPORT
+
+For now, support questions should be sent to:
+
+E<lt>netdri@dotandco.comE<gt>
+
+Please also see the SUPPORT file in the distribution.
+
+=head1 SEE ALSO
+
+E<lt>http://www.dotandco.com/services/software/Net-DRI/E<gt>
+
+=head1 AUTHOR
+
+Paulo Jorge, E<lt>paullojorgge@gmail.comE<gt>
```

Comment:
@pc-stein you might want to add your details in this file (and any other related POD files)... I haven't done anything here :p